### ISSUE TITLE
why my client 10.1 got  no match job from the server those days

### ISSUE THREAD

**Issue by l1t1** [gcp/leela-zero#673]
47 game(s) (47 self played and 0 matches) played in 1565 minutes = 1997 seconds/game, 6016 ms/move, last game took 1595
seconds.
{
    "cmd": "selfplay",
    "hash": "c83e1b6e0ffbf8e684f2d8f6261853f14c553b29ee0e70ff6c34e87d28009c43",
    "leelaz_version": "0.10",
    "options": {
        "noise": "true",
        "playouts": "1600",
        "randomcnt": "30",
        "resignation_percent": "1"
    },
    "options_hash": "16b7b7",
    "random_seed": "11386383430011194420",
    "required_client_version": "11"
}

Got new job: selfplay
net: c83e1b6e0ffbf8e684f2d8f6261853f14c553b29ee0e70ff6c34e87d28009c43.

**Comment by Mardak** [comment]
> 1997 seconds/game

That's 33 minutes per game. The server was changed in #545 to only send match games to clients that average at least 6 games an hour.

**Comment by l1t1** [comment]
@Mardak copy it

**Comment by alreadydone** [comment]
Just to be clear, the server can't yet distinguish a client (IP address?) with one fast device from a client with many slow devices, right?

**Comment by roy7** [comment]
Correct. @marcocalignano submitted an idea on server github I might implement, just haven't made time to play with it. Some other ideas have been suggested too.
